src/python/aqueduct_executor/operators/connectors/data/extract.py

```diff
@@ -16,15 +18,48 @@ class RelationalParams(models.BaseParams):
     # any user-defined tags like `{{today}}`.
     query_is_usable: Optional[bool] = False
 
-    query: str
+    query: Optional[str] = None
+    queries: Optional[List[str]] = None
 
     # TODO: Consider not including github as part of relational params when it is JSON marshalled
     github_metadata: Optional[Any]
 
-    def expand_placeholders(
-        self,
-        parameters: Dict[str, str],
-    ) -> None:
+    def _compile_chain(self, queries: List[str]) -> str:
```

Comment:
Let's add a sentence or two that describe at a high level what this function accomplishes.